docs/install-upgrade/install.md

```diff
@@ -41,19 +41,29 @@ Hedgehog has created a command line utility, called `hhfab`, that helps generate
 1. The `init` command generates a `fab.yaml` file, edit the `fab.yaml` file for your needs
     1. ensure the correct boot disk (e.g. `/dev/sda`) and control node NIC names are supplied
 1. `hhfab validate`
-1. `hhfab build`
+1. `hhfab build --mode iso`
+    1. There are two options for the format of the bootable media. ISO and IMG. An ISO is best suited to use with IPMI based virtual media. The img file is designed for writing to a removable USB drive.
```

Comment:
Please indicate the default value for the option (if the user doesn't provide it) for clarity.